AbstractDesignPatter/BridgeOfSouls .cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace AbstractDesignPatter
+{
+	public class BridgeOfSouls : IBook
+	{
+		public string getName()
```

Comment:
Please be consistent, the name of the functions should start with a capital letter.